MekHQ/src/mekhq/campaign/unit/Unit.java

```diff
@@ -3727,10 +3734,41 @@ private void calcCompositeCrew() {
             }
 
             if (entity instanceof Tank
-                    && Compute.getFullCrewSize(entity) == 1
-                    && p.hasSkill(gunType)) {
-                sumGunnery += p.getSkill(gunType).getFinalSkillValue();
-                nGunners++;
+                    && Compute.getFullCrewSize(entity) == 1) {
+                if (SkillType.isRPGGunnery(gunType) &&
+                        campaign.getGameOptions().booleanOption(OptionsConstants.RPG_RPG_GUNNERY)) {
```

Comment:
These should be in the opposite order, always check if you are using the RPG gunnery option first.